projects/element-theme/src/styles/bootstrap/_button-group.scss

```diff
@@ -80,13 +87,13 @@
     transition: none;
 
     &.btn-icon {
-      padding-inline: map.get(spacers.$spacers, 5) - 1px;
+      padding-inline: map.get(spacers.$spacers, 5) - 1px !important; // stylelint-disable-line declaration-no-important
```

Comment:
Is there no way to get rid of those important declarations?

Reply:
@dr-itz any idea why?

Reply:
yes, !important isn't needed as the ordering in the CSS is sufficient. It's probably one of the many bootstrap remains because bootstraps absolutely loves those silly !important...